packages/server/tests/localAcceptance.env

```diff
@@ -12,3 +12,4 @@ HAPI_CLIENT_TRANSACTION_RESET=50
 HAPI_CLIENT_DURATION_RESET=3600000
 HAPI_CLIENT_ERROR_RESET=[50]
 FILTER_API_ENABLED=true
+MIRROR_NODE_RETRY_DELAY = 1000
```

Comment:
Q: why is this needed?

Reply:
I was testing in CI.  It's not needed and removed.